app/src/main/java/org/simple/clinic/newentry/PatientEntryScreenController.kt

```diff
@@ -217,13 +163,12 @@ class PatientEntryScreenController @Inject constructor(
         .ofType<PatientStateTextChanged>()
         .map { it.state }
 
-    val addressChanges = Observables.combineLatest(colonyOrVillageChanges, districtChanges, stateChanges)
-    { colonyOrVillage, district, state ->
-      OngoingPatientEntry.Address(colonyOrVillage.toNullable().nullIfBlank(), district, state)
-    }
+    val addressChanges = Observables
+        .combineLatest(colonyOrVillageChanges, districtChanges, stateChanges) { colonyOrVillage, district, state ->
+          Address(colonyOrVillage, district, state)
+        }
 
-    return Observables.combineLatest(personDetailChanges, phoneNumberChanges, addressChanges)
-    { personal, phone, address ->
+    return Observables.combineLatest(personDetailChanges, phoneNumberChanges, addressChanges) { personal, phone, address ->
       OngoingPatientEntryChanged(OngoingPatientEntry(personal, address, phone.toNullable()))
```

Comment:
`phone.toNullable()` can also go away if you decide to remove `Optional` from `phoneNumberChanges`.

Reply:
Phone number is still optional. We just don't have an explicit option to make it nullable.

Reply:
TODO: Add an assertion for creating a `PhoneNumber` instance with a blank phone number.